src/routes/profiles.py

```diff
@@ -0,0 +1,41 @@
+from apiflask import APIBlueprint, abort, auth_required, doc, input, output
+
+import src.controllers.profiles as c
+import src.models.profiles as m
+from ..utils.auth import auth
+from ..utils.enums import ControllerStatus
+
+router = APIBlueprint("prof", __name__, "Profiles", url_prefix="/api")
+
+# Get no endpoint para perfil, dentro de la propiedad *arriba
+# Random note over there, just leave it behind
+
+
+@router.get("/profile")
+@output(m.ProfileData(many=True), 200)
```

Comment:
No, en esta ruta sólo queremos que regrese un solo resultado
```suggestion
@output(m.ProfileData, 200)
```

Reply:
Ah sí, again para pruebas. My bad